kube/pod_test.go

```diff
@@ -796,87 +1718,176 @@ func TestPodPreFlight(t *testing.T) {
 			containers:
 			-
 				name: pre-role
+				resources:
+					requests:
+						memory: 128Mi
+					limits:
+						memory: 384Mi
 			restartPolicy: OnFailure
 			terminationGracePeriodSeconds: 600
-	`, "\t", "    ", -1)
-	if !assert.NoError(yaml.Unmarshal([]byte(expectedYAML), &expected)) {
-		return
-	}
-
-	testhelpers.IsYAMLSubset(assert, expected, actual)
+	`, actual)
 }
 
-func TestPodPostFlight(t *testing.T) {
+func TestPodMemoryHelmDisabled(t *testing.T) {
 	assert := assert.New(t)
-	role := podTestLoadRole(assert, "post-role")
+	role := podTestLoadRole(assert, "pre-role")
 	if role == nil {
 		return
 	}
-
 	pod, err := NewPod(role, ExportSettings{
-		Opinions: model.NewEmptyOpinions(),
+		CreateHelmChart: true,
+		Repository:      "theRepo",
+		Opinions:        model.NewEmptyOpinions(),
+		UseMemoryLimits: true,
```

Comment:
Why ? Because of `Disabled` in the test name ? We are testing here what happens when the helm chart has support for memory/cpu limits per fissile and the user disables it in the values.yml.

Or am I misunderstanding the comment completely?

Reply:
Actually it does matter. For `UseMemoryLimits = false` fissile produces a helm chart where the user cannot do anything at runtime about mem limits, neither enable, nor disable. Ditto cpu limits. IIRC that is as we wanted it. Definitely out of scope for the test suite work.